cmd/charm/charmcmd/list.go

```diff
@@ -71,24 +73,28 @@ func (c *listCommand) Run(ctxt *cmd.Context) error {
 	}
 	defer client.jar.Save()
 
-	if c.user == "" {
+	if c.users == "" {
 		resp, err := client.WhoAmI()
 		if err != nil {
 			return errgo.Notef(err, "cannot retrieve identity")
 		}
-		c.user = resp.User
+		c.users = resp.User
 	}
-
-	err = validateNames([]string{c.user})
+	users := strings.Split(c.users, ",")
+	err = validateNames(users)
 	if err != nil {
 		return errgo.Mask(err)
 	}
-
-	path := "/list?owner=" + c.user + "&sort=name,-series"
+	v := url.Values{}
+	v.Set("sort", "name,-series")
+	for _, user := range users {
+		v.Add("owner", user)
+	}
+	path := "/list?" + v.Encode()
 	var resp params.ListResponse
 	err = client.Get(path, &resp)
 	if err != nil {
-		return errgo.Notef(err, "cannot list for user %s", path)
+		return errgo.Notef(err, "cannot list for user(s) %s", path)
```

Comment:
Why include the whole path here rather than just the users as the message implies?